### PR TITLE
Chore: Replaces `Hero` FSUI

### PR DESCRIPTION
**Pull request by renatamottam**
> `@faststore/ui` package needs to be updated after merge of https://github.com/vtex/faststore/pull/1556

## What's the purpose of this pull request?
- [x] Uses `Hero` from the `faststore/ui`
- [x] Remove `Hero` folder

## How does it work?
Everything should look as before. (Visually)

## How to test it?
You can check the Homepage and PLP

### CODE REVIEW

File: src/components/sections/Hero/Hero.tsx [hunk 1]
```diff
@@ -3,17 +3,9 @@ import {
...
-import Icon from 'src/components/ui/Icon'
-import Image from 'src/components/ui/Image/Image'
-
 import Section from '../Section'
-import styles from './hero.module.scss'
-
-type Variant = 'primary' | 'secondary'
-type ColorVariant = 'main' | 'light' | 'accent'
 
 export interface HeroProps {
```
Comment (lucasfp13) [inline src/components/sections/Hero/Hero.tsx:10]: We need to extend `HeroProps` type (from `@faststore/ui`) here

Comment (eduardoformiga) [reply src/components/sections/Hero/Hero.tsx:10]: nice catch!

**Review by eduardoformiga — COMMENTED** [review]
Looks good.
LH is complaining about the hero image, though. We need to deep dive into this.
Screen Shot 2022-12-16 at 12 30 58

**Review by eduardoformiga — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -31,10 +31,10 @@
     "@envelop/parser-cache": "^2.2.0",
     "@envelop/validation-cache": "^2.2.0",
     "@faststore/api": "^2.0.3-alpha.0",
-    "@faststore/components": "^2.0.18-alpha.0",
+    "@faststore/components": "2.0.19-alpha.0",
```
Comment (hellofanny) [inline package.json:34]: should we take the opportunity and also update it to latest version (2.0.29-alpha.0)?

File: src/components/sections/ProductDetails/ProductDetails.tsx [hunk 3]
```diff
@@ -125,11 +125,7 @@ function ProductDetails({ context: staleProduct }: Props) {
           <ProductTitle
             title={<h1>{name}</h1>}
             label={
-              <UIDiscountBadge
-                listPrice={listPrice}
-                spotPrice={lowPrice}
-                size="big"
-              />
+              <UIDiscountBadge listPrice={listPrice} spotPrice={lowPrice} />
```
Comment (hellofanny) [inline src/components/sections/ProductDetails/ProductDetails.tsx:128]: Puts `size` prop back when https://github.com/vtex/faststore/pull/1570 merged.

**Review by hellofanny — APPROVED** [review]

**Review by ArthurTriis1 — APPROVED** [review]

**Pull request merged**